data/snippets.conf

```diff
@@ -58,6 +66,11 @@ try=try%block%\ncatch (%cursor%)%block_cursor%
 if=if (%cursor%)%block_cursor%
 else=else%block_cursor%
 for=for (int i = 0; i < %cursor%; i++)%brace_open%\n%brace_close%
+syso=System.out.println("%cursor%");
+Syso=System.out.println("%cursor%");
+main=package %cursor%;\n\npublic class Main {\n\tpublic static void main(String[] args) {\n\t\t%cursor%\n%brace_close1%%brace_close%
+class=public class %cursor% {\n\t%cursor%\n}
+junit=import org.junit.*;\n \npublic class Test%cursor% {\n\t//~ @BeforeClass\n\t//~ public static void setUpClass() throws Exception {\n\t\t//~ \n\t//~ }\n\n\t//~ @Before\n\t//~ public void setUp() throws Exception {\n\t\t//~ \n\t//~ }\n\n\t@Test\n\tpublic void test() {\n\t\t\n\t}\n\n\t//~ @After\n\t//~ public void tearDown() throws Exception {\n\t\t//~ \n\t//~ }\n\t\n\t//~ @AfterClass\n\t//~ public static void tearDownClass() throws Exception {\n\t\t//~ \n\t//~ }\n\t\n}
```

Comment:
I think uncomment is not a problem with Geany, I think it's very fast to just select and Ctrl+E line to uncomment, as you can see it's Geany comment not normal comment. And in JUnit test @Before is common, but @BeforeClass, @After, @AfterClass not very common. Maybe just remove @BeforeClass, @After, @AfterClass part in this case and uncomment @Before part ?